components/raftstore-v2/src/batch/store.rs

```diff
@@ -436,6 +454,22 @@ impl<EK: KvEngine, ER: RaftEngine> StoreRouter<EK, ER> {
     pub fn logger(&self) -> &Logger {
         &self.logger
     }
+
+    #[inline]
+    pub fn send_read_command(
```

Comment:
I don't think this abstraction is useful.

Reply:
how do you reuse the code then? It's called in multiple places. 
BTW, there's the similar one in v1.

Reply:
Like where? It should only be called once.

Reply:
It's to hide some internal details like PeerMsg::RaftQuery or PeerMsg::RaftCommand. Also it's more convenient to use.